### ISSUE TITLE
Need help with @part

### ISSUE THREAD

**Issue by CC007** [manifold-systems/manifold#476]
I'm having issues with getting `@part` to work. [Here is my code](https://github.com/CC007/ManifoldDemo/blob/master/src/main/java/com/github/cc007/Demo5.java).

When `@part` is commented out, the code compiles fine. the `getName()` call in `Car.start()` and `Car.accelerate(...)` aren't calling `Tesla.getName()`, but this was expected, because of the missing `@part` annotation.

When I try to enable the `@part` annotation, I get the following error:
```
java: variable $theSelf is already defined in class com.github.cc007.Demo5.Car
java: variable $interfacesFullyCovered is already defined in class com.github.cc007.Demo5.Car
```

What causes this issue and how can I resolve it? Could it be that multiple Manifold annotation processors are trying to do the same thing? [Here is my pom.xml file](https://github.com/CC007/ManifoldDemo/blob/master/pom.xml).

**Comment by CC007** [comment]
This happens, regardless if I compile with maven or Intellij itself.

Manifold version: 2023.1.15
Java version: openjdk 20.0.2
Manifold IntelliJ plugin version: 2023.1.15
IntelliJ version: 2023.1.5

**Comment by rsmckinney** [comment]
Just finished looking at this, it's a simple fix on my end that deals with annotation processor (lombok) combined with manifold, which is not an annotation processor. This will be fixed by EOD tomorrow.

For now, you can work around this simply be removing lombok from your demo project. It looks like you are only using it for generating two constructors; easy to fix. But, if you can wait, it will be fixed in the new release coming out tomorrow evening.

Thanks for reporting this!

**Comment by CC007** [comment]
Thanks for the quick reaction! I've been using Lombok for a long time now and only recently learned of Manifold. The feature set of these two seem to complement each other very nicely!

I don't mind waiting a day or 2. At least I don't have to wait months like with a couple of my spring-cloud tickets. 😄 They seem to be pretty busy over there.

**Comment by rsmckinney** [comment]
Fix available with release 2023.1.18. Thanks for reporting this!
